examples/AdminClient/Program.cs

```diff
@@ -84,7 +95,205 @@ static async Task CreateTopicAsync(string bootstrapServers, string topicName)
                 }
                 catch (CreateTopicsException e)
                 {
-                    Console.WriteLine($"An error occured creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
+                    Console.WriteLine($"An error occurred creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
+                }
+            }
+        }
+
+        static List<AclBinding> ParseAclBindings(string[] args, bool many)
+        {
+            var numCommandArgs = args.Length;
+            if (
+                many ?
+                (numCommandArgs == 0 || numCommandArgs % 7 != 0) :
+                numCommandArgs != 7
+            )
```

Comment:
i would format this: 

```
if (many
           ? (numCommandArgs == 0 || numCommandArgs % 7 != 0)
           : numCommandArgs != 7)
{
```

i'd actually change some other things about this but it's being picky and not worth worrying about.

Reply:
Done